registration/tests/test_approval.py

```diff
@@ -38,23 +38,52 @@ class ApprovalTestCase(TestCase):
     """Tests for the helper functions in the approval module."""
 
     def test_accept_application(self):
```

Comment:
Try to split this test, it's testing too much.

Reply:
👍